<launch>
  <!-- Launch a map from the maps folder-->
  <arg name="map" default="$(find range_entropy)/maps/classroom_incomplete.yaml"/>
  <node pkg="map_server" name="map_server" type="map_server" args="$(arg map)"/>

  <!-- Launch the visualizer -->
  <node pkg="range_entropy" name="entropy_visualizer" type="entropy_visualizer" output="screen">
    <rosparam command="load" file="$(find range_entropy)/launch/params.yaml"/>
  </node>

</launch>
